images/release/golang-1.8/Dockerfile

```diff
@@ -31,6 +32,6 @@ RUN mkdir $TMPDIR && \
     git config --system user.name origin-release-container && \
     git config --system user.email none@nowhere.com
 
-WORKDIR /go/src/github.com/openshift/origin
+WORKDIR $WORKDIR
```

Comment:
I'm not sure why you would parameterize workdir - that's not something that images expect to change, and we already have knobs for this.

I guess - why are you changing the images?